android/titanium/src/java/org/appcelerator/titanium/TiC.java

```diff
@@ -1447,6 +1447,11 @@
 	/**
 	 * @module.api
 	 */
+	public static final String PROPERTY_FREE_SPACE = "freeSpace";
```

Comment:
I don't like the property-name. What about a more "boolean-like" property, e.g. `showSpaceWhenHidden` or `truncateSpace`? Or even expose the used constant directly and introduce a `hiddenBahavior` property. That keeps it more flexible, allows more configuration and sounds more native. Thanks!